docs/en/Blog-Posts/2021-04-05 CmsKit/POST.md

```diff
@@ -28,13 +28,13 @@ If you're using the open-source version or adding the module to an existing proj
 You can run the following command to add the module to an open-source solution. 
 
 ```bash
-abp add-module Volo.CmsKit
+abp add-module Volo.CmsKit --skip-db-migrations true
```

Comment:
```suggestion
abp add-module Volo.CmsKit --skip-db-migrations
```

this should work. no need to `true`

Reply:
OK, but with this change it will be, right? So, you should write `--skip-db-migrations true` to rel-4.3 branch, `--skip-db-migrations` to dev branch in this case.

Reply:
`--skip-db-migrations true` will work for both. So i didn't mind.